### ISSUE TITLE
ERP plot - saving the image

### ISSUE THREAD

**Issue by jrliu95** [luiztauffer/ecogVIS#18]
Clicking the "Save image" button didn't seem to do anything, can you have it open up a dialog box where the user can select where to save it to?

**Comment by luiztauffer** [comment]
If you right-click any item in the grid, it will open a list of options, including exporting as image.
This pyqtgraph solution already seemed quite complete and great, in my opinion, and what I want to do is being able to directly open the 'export image' dialog by button click. This has been a bit tricky to do, and it's not a priority, since it can be done by mouse-clicking, but it's on the plan!
Try the mouse-click option and let me know what you think of it?

**Comment by jrliu95** [comment]
Oh, okay. Yeah, having it on the button would be ideal but doing the right click is also fine.

When I tried it, it seemed like it was only going to save the one plot that I double clicked over? But also when it brought up the dialog box with export options, it wouldn't let me choose anything, just kept giving that error noise when I clicked anywhere in the dialog box.

![Screen Shot 2019-06-20 at 1 36 58 PM](https://user-images.githubusercontent.com/17503400/59879851-8809d500-9360-11e9-9fa6-7521b8aa54c1.png)
